src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -485,6 +487,52 @@ def get_recordio_protobuf_dmatrix(path, is_pipe=False):
         raise exc.UserError("Failed to load recordio-protobuf data with exception:\n{}".format(e))
 
 
+def _get_pipe_mode_files_path(data_path: Union[List[str], str]) -> List[str]:
+    """
+    :param data_path: Either directory or file
+    """
+    # For pipe mode, we leverages mlio directly by creating a list of SageMakerPipe.
+    if isinstance(data_path, list):
+        files_path = data_path
+    else:
+        files_path = [data_path]
+        if not os.path.exists(f"${data_path}_0"):
+            logging.info('Pipe path {} does not exist!'.format(data_path))
+            return None
+    return files_path
+
+
+def _get_file_mode_files_path(data_path: Union[List[str], str]) -> List[str]:
+    """
+    :param data_path: Either directory or file
+    """
+    # In file mode, we create a temp directory with symlink to all input files or
+    # directories to meet XGB's assumption that all files are in the same directory.
+
+    if isinstance(data_path, list):
+        logging.info('File path {} of input files'.format(data_path))
```

Comment:
```suggestion
        logging.info("File path {data_path} of input files")
```

Reply:
Is this for debugging? Not sure this log provides a lot of value here so consider removing it or updating the message to be a little more descriptive.

Reply:
This can be removed now that we have warning and validation logic in place